src/Microsoft.Framework.ApplicationHost2/project.json

```diff
@@ -51,12 +55,12 @@
                 "System.Collections": "4.0.10-beta-*",
                 "System.Diagnostics.Tools": "4.0.0-beta-*",
                 "System.Diagnostics.Debug": "4.0.10-beta-*",
+                "System.Diagnostics.TraceSource": "4.0.0-beta-*",
```

Comment:
Are we moving TraceSource stuff out of M.F.Logging? It's not on the queue from the API review. Unfortunately, due to source-inclusion, we have to have this package here even if we don't use it, unless the TraceSource logger is moved out of M.F.Logging and into a separate package.